proxy/HostStatus.cc

```diff
@@ -57,3 +101,28 @@ HostStatus::getHostStatus(const char *key)
 
   return static_cast<HostStatus_t>(_status);
 }
+
+void
+HostStatus::createHostStat(const char *name)
+{
+  InkHashTableEntry *entry;
+  entry = ink_hash_table_lookup_entry(hosts_stats_ids, name);
+  if (entry == nullptr) {
+    RecRegisterRawStat(host_status_rsb, RECT_PROCESS, (stat_prefix + name).c_str(), RECD_INT, RECP_NON_PERSISTENT, (int)host_count,
+                       RecRawStatSyncSum);
+    Debug("host_statuses", "HostStatus::createHostStat():  key: %s", name);
+    ink_hash_table_insert(hosts_stats_ids, name, reinterpret_cast<void *>(host_count));
+    setHostStatus(name, HostStatus_t::HOST_STATUS_UP);
+    host_count++;
```

Comment:
It took me a while to realize that `host_count` is the index of the next stat ID. Suggest that you name it `next_stat_id`.